[Pre-Commit] Running on (#3323)
demisto/demisto-sdk: CHANGELOG.md

* Fixed an issue where **validate** would not properly detect dependencies of core packs.
* Removed usages of Random in unit tests to ensure the tests are deterministic.
* **validate** will now run on all the pack content items when the pack supported marketplaces are modified.

> * Fixed an issue where **pre-commit** running when there was no change to the files

Reviewer: ```suggestion
* **pre-commit** will stop running at the start when there was no change to any files.
```
